synthtool/gcp/templates/python_notebooks_testing_pipeline/.github/workflows/linter/requirements.txt

```diff
@@ -0,0 +1,9 @@
+git+https://github.com/tensorflow/docs
+ipython
+jupyter
+nbconvert
+black==20.8b1
+pyupgrade==2.7.3
+isort==5.6.4
+flake8==3.9.0
+nbqa==0.6.0
```

Comment:
to avoid the issue from https://github.com/nbQA-dev/nbQA/issues/657 , I'd suggest updating to `1.1.0`

Reply:
Updated in latest commit!